components/omega/doc/design/OmegaV0ShallowWater.md

```diff
@@ -0,0 +1,607 @@
+= -g\nabla(h_i+b_i) - \nabla K_i + \nu_2 \nabla^2 u_e - \nu_4 \nabla^4 u_e + \mathcal{D}_e + \mathcal{F}_e
+\hspace{1cm}   (4)
+$$
+
+$$
+\frac{\partial h_i}{\partial t} + \nabla \cdot \left([h_i]_e u_e\right) = 0,
+\hspace{1cm}   (5)
+$$
+
+$$
+\frac{\partial h_i \phi_i}{\partial t} + \nabla \cdot \left(u_e [h_i \phi_i]_e \right) = \kappa_2 h_i \nabla^2 \phi_i - \kappa_4 h_i \nabla^4 \phi_i,
+\hspace{1cm}   (6)
+$$
+
+where subscripts $i$, $e$, and $v$ indicate cell, edge, and vertex locations ($i$ was chosen for cell because $c$ and $e$ look similar).  Here square brackets $[\cdot]_e$ and $[\cdot]_v$ are quantities that are interpolated to edge and vertex locations. The interpolation is typically centered, but may vary by method, particularly for advection schemes. For vector quantities, $u_e$ denotes the normal component at the center of the edge, while $u_e^\perp$ denotes the tangential component. In the discrete system, the normal component $u_e$ points positively from the lower cell index to the higher cell index, while the tangential component $u_e^\perp$ points positively $90^o$ to the left of $u_e$ (for unit vectors, ${\bf n}_e^\perp = {\bf k}\times {\bf n}_e$).
+
+The discretized momentum drag and forcing terms are
+$$
+\mathcal{D}_e = - Ra \, u_e - C_D \frac{u_e\left|u_e\right|}{[h_i]_e}
+$$
+
+$$
+\mathcal{F}_e =  C_W \frac{(u_W - u_e)\left|u_W - u_e\right|}{[h_i]_e}
+$$
+
+### 3.2 Variable Definitions
+
+Table 1. Definition of variables
+
+| symbol  | name             | units    | location | name in code | notes  |
+|---------------------|-----------------------------|----------|-|---------|-------------------------------------------------------|
+| $b$                 | height of bottom (pos. up)  | m        | cell     |   |                                                              |
+| $C_D$               | bottom drag                 | 1/m      | constant |   |                                                              |
+| $C_W$               | wind stress coefficient     | 1/m      | constant |   |                                                              |
+| $\mathcal{D} $      | drag                        | m/s^2    | edge     |   |                                                              |
+| $f$                 | Coriolis parameter          | 1/s      | vertex   | fVertex  |                                                              |
+| $\mathcal{F} $      | forcing                     | m/s^2    | edge     |   |                                                              |
+| $g$                 | gravitational acceleration  | m/s^2    | constant |   |                                                              |
+| $h$                 | thickness of layer          | m        | cell     | layerThickness  |                                                              |
```

Comment:
OMEGA variable names start with a capital letter

Reply:
Thanks. I made that change. The variable names really go in the design section below, but it is so convenient to put here that I started to fill it in.